src/App.js

```diff
@@ -1,12 +1,24 @@
 /* eslint-disable react/prefer-stateless-function */
 import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Home from './components/Home';
+import Nav from './components/Nav';
+import Quotes from './components/Quotes';
 
 class App extends React.Component {
   render() {
     return (
       <div className="App">
-        <Calculator />
+        <Router>
+          <Nav />
+          <Routes>
+            <Route path="/home" element={<Home />} />
```

Comment:
- [ ] It will be better if you change the path from `/home` to `/`, so it will be the default page when the app starts.

Reply:
Hi, @omarsalem7, Thank you for the reviewed feedback. I have implemented your suggestion.